tests/test_ote_training.py

```diff
@@ -271,6 +276,57 @@ def _run_ote_training(self, data_collector):
         data_collector.log_final_metric("metric_value", score_value)
 
 
+class ClassificationTestNNCFAction(OTETestNNCFAction):
+    _name = "nncf"
+    _depends_stages_names = ["training"]
+
+    def _run_ote_nncf(
+        self, data_collector, model_template, dataset, trained_model, environment
+    ):
+        logger.debug("Get predictions on the validation set for exported model")
+        self.environment_for_nncf = deepcopy(environment)
+
+        logger.info("Create NNCF Task")
+        nncf_task_class_impl_path = model_template.entrypoints.nncf
+        if not nncf_task_class_impl_path:
+            pytest.skip("NNCF is not enabled for this template")
+
+        if not is_nncf_enabled():
+            pytest.skip("NNCF is not installed")
+
+        logger.info("Creating NNCF task and structures")
+        self.nncf_model = ModelEntity(
+            dataset,
+            self.environment_for_nncf.get_model_configuration(),
+        )
+        self.nncf_model.set_data("weights.pth", trained_model.get_data("weights.pth"))
+        
+        if 'aux_model_1.pth' in trained_model.model_adapters:
+            self.nncf_model.set_data("aux_model_1.pth", trained_model.get_data("aux_model_1.pth"))
```

Comment:
@pfinashx , are you sure that only this name is always used?

Reply:
Yes, at least for Mobilenet models.

Reply:
Yep, at least for Mobilenet model